pkg/pravega/pravega_segmentstore.go

```diff
@@ -188,6 +222,40 @@ func makeCacheVolumeClaimTemplate(pravegaSpec *api.PravegaSpec) []corev1.Persist
 	}
 }
 
+func makeSegmentstorePodDisruptionBudget(pravegaCluster *api.PravegaCluster) *policyv1beta1.PodDisruptionBudget {
+	var maxUnavailable intstr.IntOrString
+
+	if pravegaCluster.Spec.Pravega.SegmentStoreReplicas == int32(1) {
```

Comment:
Same comment about min available and max unavailable.

Reply:
Please check response in controller comment.